tests/monitor/monitor.test.mjs

```diff
@@ -36,6 +36,30 @@ describe('setup', () => {
     await compose.down();
   });
 
+  describe('handlers', () => {
+    it('should always have correct number of handlers', async () => {
+      const expectedHandlerCount = 50; // TODO: update this if more handlers are added
+      const { Handlers: handlerList } = await io.getInfo();
+      // assert _eval, _default and prune are first
+      assert.ok(
+        handlersList.indexOf('_eval') === 0,
+        '_eval should be first handler',
+      );
+      assert.ok(
+        handlersList.indexOf('_default') === 1,
+        '_default should be second handler',
+      );
+      assert.ok(
+        handlersList.indexOf('prune') === 2,
+        'prune should be third handler',
+      );
```

Comment:
these will fail until we get clarification from AO team if the following handlers are still necessary after upgrading to aos 2.0.0

```
{
  {
     handle = function: 0x912128,
     pattern = function: 0xd37708,
     name = "Assignment-Check"
  },
  {
     handle = function: 0x755910,
     pattern = function: 0x929600,
     name = "sec-patch-6-5-2024"
  },
```

Reply:
<img width="1041" alt="image" src="https://github.com/user-attachments/assets/90ef5d96-66a5-4e4f-b003-091eacb0bc8b">

Reply:
https://discord.com/channels/1210396395643863101/1210403116361580574/1288924014718357514